lib/impl/AzureBlobFiles.js

```diff
@@ -73,7 +73,7 @@ class AzureBlobFiles extends Files {
   constructor (credentials, tvm) {
     super()
     this.tvm = tvm
```

Comment:
minor comment let's define all those as private in the jsdoc, so we don't expose them in autocompletion:
```
/** @private */
this.tvm
```
same for this.hasOwnCredentials and this.crendentials